### PR TITLE
Add linting to foundation-demo

### PR DESCRIPTION
**Pull request by lderrickable**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Issue Number: 
https://github.com/graycoreio/daffodil/issues/161
https://github.com/graycoreio/daffodil/issues/106

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by damienwebdev** [comment]
@lderrickable could we change all the "wrappers" used in tests to "WrapperComponent" and still pass linting?

**Comment by lderrickable** [comment]
> @lderrickable could we change all the "wrappers" used in tests to "WrapperComponent" and still pass linting?

Yeah, I actually really liked that, but I was planning doing it in the future so this PR didn't get unnecessarily large. I can do it here if you'd prefer though

**Comment by damienwebdev** [comment]
Since we're touching them here, we may as well.

**Comment by lderrickable** [comment]
> Since we're touching them here, we may as well.

Done

File: apps/foundation-demo/src/app/product/components/add-to-cart/add-to-cart.component.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 @Component({
   selector: 'demo-add-to-cart',
   templateUrl: './add-to-cart.component.html',
   styleUrls: ['./add-to-cart.component.scss'],
-  host: {'class': 'add-to-cart'},
   encapsulation: ViewEncapsulation.None
 })
 export class AddToCartComponent {
+  @HostBinding('class') class = 'add-to-cart';
```
[the remark this answers is not in the document]
Comment (lderrickable) [reply apps/foundation-demo/src/app/product/components/add-to-cart/add-to-cart.component.ts]: There a long explanation that would explain this, but let's just say I changed it back

**Pull request merged**